src/schemas/components/SchemaFormProperties.vue

```diff
@@ -1,16 +1,7 @@
 <template>
   <p-content class="schema-form-properties">
     <template v-for="[key, property] in properties" :key="key">
-      <template v-if="isPropertyWith(property, '$ref')">
-        <SchemaFormProperty
-          :value="getValue(key)"
-          :property="getSchemaDefinition(schema, property.$ref)"
-          :required="getRequired(key)"
-          @update:value="setValue(key, $event)"
-        />
-      </template>
```

Comment:
This got moved into the SchemaFormPropertyInput component where it now does some more complete resolving of references.